driver/src/main/java/oracle/nosql/driver/values/JsonUtils.java

```diff
@@ -766,23 +766,45 @@ public static String convertBytesToHex(byte[] byteArray) {
      * quotes (").
      */
     public static void validateJsonConstruct(String jsonInput) {
```

Comment:
This method comes from the nson repo. I just sync'd them up yesterday. Change should be made there first. I'd prefer if the method were to take an optional JsonOptions object, allowing the caller to decide about validation